### PR TITLE
Feature/add infinite loader

### PR DESCRIPTION
**Pull request by JustinCast**
# Add Infinite Registry Table Loader
### What does this PR do?
Add infinite registry table loader

## How should this be tested?
run yarn storybook
go to http://localhost:9009 and click on InfiniteRegistryTableLoader

### CODE REVIEW

File: src/InfiniteRegistryTableLoader.js [hunk 1]
```diff
@@ -0,0 +1,185 @@
+//#region imports
+import React from 'react'
+import InfiniteLoader from 'react-virtualized/dist/commonjs/InfiniteLoader'
+import { Table, Column } from 'react-virtualized/dist/commonjs/Table'
+import { makeStyles } from '@material-ui/styles'
+import PropTypes from 'prop-types'
+import TableCell from '@material-ui/core/TableCell'
+import clsx from 'clsx'
+//#endregion
```
Comment (tetogomez) [inline src/InfiniteRegistryTableLoader.js:9]: this kind of comment is not necessary

File: src/InfiniteRegistryTableLoader.js [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+  rowHeight,
+  headerHeight
+}) => {
+  const classes = useStyles()
+
+  const cellRenderer = ({ cellData, columnIndex }) => {
+    return (
+      <TableCell
+        component='div'
+        // onClick={columnIndex === 4 ? () => handleClickOpen() : undefined}
```
Comment (tetogomez) [inline src/InfiniteRegistryTableLoader.js:70]: we shouldn't code commented, delete it if you won't use it

File: src/stories/6-InfiniteRegistryTableLoader.stories.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+import IconButton from '@material-ui/core/IconButton'
+import { AutoSizer } from 'react-virtualized'
+
+export default {
+  title: 'InfiniteRegistryTableLoader',
+  component: InfiniteRegistryTableLoader
+}
+
+export const RegistryLoader = () => {
+  //#region samples
```
Comment (tetogomez) [inline src/stories/6-InfiniteRegistryTableLoader.stories.js:13]: same here, delete it!

**Comment by JustinCast** [comment]
@xavier506 @tetogomez are there something else I missed?

**Pull request merged**